go/vt/vtgate/tx_conn.go

```diff
@@ -62,6 +59,16 @@ var txAccessModeToEOTxAccessMode = map[sqlparser.TxAccessMode]querypb.ExecuteOpt
 	sqlparser.ReadOnly:               querypb.ExecuteOptions_READ_ONLY,
 }
 
+type commitPhase int
+
+const (
+	COMMIT2PC_CREATETRANSACTION = iota
+	COMMIT2PC_PREPARE           = iota
+	COMMIT2PC_STARTCOMMIT       = iota
+	COMMIT2PC_PREPARECOMMIT
+	COMMIT2PC_CONCLUDE
+)
```

Comment:
You probably meant to use the type, and `iota` should generally only be used on the first value.

```suggestion
const (
	Commit2pcCreateTransaction commitPhase = iota
	Commit2pcPrepare
	Commit2pcStartCommit
	Commit2pcPrepareCommit
	Commit2pcConclude
)```